lib/services/datafactoryManagement/lib/models/index.d.ts

```diff
@@ -4040,7 +4392,7 @@ export interface ScheduleTriggerRecurrence {
  * occurrences.
  */
 export interface ScheduleTrigger extends MultiplePipelineTrigger {
-  recurrence?: ScheduleTriggerRecurrence;
+  recurrence: ScheduleTriggerRecurrence;
```

Comment:
Making an optional property into a required property is breaking change. This requires a major version increment.